### ISSUE TITLE
Discuss: Switch polkadot branch naming to `release-polkadot-v1.x.x`

### ISSUE THREAD

**Issue by haerdib** [scs/substrate-api-client#661]
Parity now uses a different polkadot branch naming system: instead of `polkadot-v0.x.x` it's now `release-polkadot-v1.x.x`. I think it probably makes sense to follow parity in the new naming system with our branches as well.

@Niederb @masapr  what do you think?

**Comment by Niederb** [comment]
@haerdib I agree: I would also follow the naming system from polkadot. Otherwise it gets confusing

**Comment by haerdib** [comment]
I pushed the existing branches `polkadot-v1.x.x` with a new name `release-polkadot-v1.x.x`. I'd rather not remove the already pushed `polkadot-v1.x.x` branches. For the future, however, let's use only `release-polkadot-v1.x.x` branches.
